Sources/Plasma/FeatureLib/pfConsole/pfConsoleCommandsNet.cpp

```diff
@@ -544,7 +544,7 @@ PF_CONSOLE_CMD( Net,        // groupName
                "returns the age of the age" )   // helpString
 {
     PrintStringF(PrintString, "Current age is %s, elapsed time since birth = %f secs", 
-        NetCommGetAge()->ageDatasetName, plNetClientMgr::GetInstance()->GetCurrentAgeElapsedSeconds());
+        NetCommGetAge()->ageDatasetName.c_str(), plNetClientMgr::GetInstance()->GetCurrentAgeElapsedSeconds());
```

Comment:
Vaguely related, I've created issue #479 for porting the whole thing to plString/plFormat